src/demos/demos.css

```diff
@@ -6,7 +6,29 @@ body {
 .example-container {
   padding: 0 24px 24px;
 }
+@media (min-width: 1025px) {
```

Comment:
A background color does help see where the component ends, but I'm not sure if a restricted container inside of the window helps.

Also feel free to access our color vars like var(--calcite-app-background-content) for the general gray bg-color.

Reply:
I do agree that setting a max-width on the `example-container` would be good, but we can probably set it's width to a reasonable vw value or something.

Reply:
Used some calcite colors. There a specific width you were thinking?
I'm happy to adjust this - the aim is to give a frame of reference for the components, and the gist I got from the inline widths used in so many demos was full width of the screen was too wide.